### DISCUSSION TITLE
Using CommunityToolkit.WinUI.UI.Animations with the Windows head.

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#11211] [Q&A]
I'm trying to use CommunityToolkit.WinUI.UI.Animations in the Windows project. So I've added the CommunityToolkit.WinUI.UI.Animations package to the project:

```
<ItemGroup Condition="'$(TargetFramework)' == 'net7.0-windows10.0.18362'">
	<PackageReference Include="CommunityToolkit.WinUI">
		<Version>7.1.2</Version>
	</PackageReference>
	<PackageReference Include="CommunityToolkit.WinUI.UI">
		<Version>7.1.2</Version>
	</PackageReference>
	<PackageReference Include="CommunityToolkit.WinUI.UI.Animations">
		<Version>7.1.2</Version>
	</PackageReference>
	<PackageReference Include="WinUIEx">
		<Version>1.8.0</Version>
	</PackageReference>
</ItemGroup>
```

But I'm getting an error when I try to use it:

```
using CommunityToolkit.WinUI.UI.Animations;

#if NET6_0_OR_GREATER && WINDOWS && !HAS_UNO
    public void SetListDataItemForNextConnectedAnimation(object item) => Frame.SetListDataItemForNextConnectedAnimation(item);
#else
    public void SetListDataItemForNextConnectedAnimation(object item) { return; }
#endif
```

>The type or namespace name 'WinUI' does not exist in the namespace 'CommunityToolkit' (are you missing an assembly reference?)	X3_Produccio_UNO (net7.0-windows10.0.18362)

The error points to the using sentence.

**Comment by jeromelaban** [comment]
This would likely mean that you have not included `CommunityToolkit.WinUI.UI.Animations` as well, similar to [this documentation](https://platform.uno/docs/articles/uno-community-toolkit.html?tabs=tabid-winui#referencing-the-windows-community-toolkit-from-a-cross-targeted-library).

**Reply by mcNets** [reply]
I started a new project, followed the same steps and now it compiles. 🤷‍♂️

Maybe I'm wrong, but shouldn't the docs about [CommunityToolkit.MVVM](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/DependencyInjection/HowTo-CommunityToolkit.html?q=mvvm) be updated? Is it still necessary to use unoapp-extensions?
